sdc.go

```diff
@@ -55,6 +55,43 @@ func (s *System) GetSdc() ([]types.Sdc, error) {
 	return sdcs, nil
 }
 
+// GetSdcByID returns a Sdc searched by id
+func (s *System) GetSdcByID(id string) (*Sdc, error) {
+	defer TimeSpent("GetSdcByID", time.Now())
+
+	path := fmt.Sprintf("api/instances/Sdc::%v", id)
+
+	var sdc types.Sdc
+	err := s.client.getJSONWithRetry(
+		http.MethodGet, path, nil, &sdc)
+	if err != nil {
```

Comment:
Here also the same comments as mentioned by @khareRajshree in ChangeSdcName fn. when err is not nil it should return the error

Reply:
Changed and pushed fresh.